services/core/ForwardHistorian/forwarder/agent.py

```diff
@@ -295,7 +296,11 @@ def historian_setup(self):
                 _log.debug(
                     "Setting up to forward to {}".format(destination_vip))
                 event = gevent.event.Event()
-                agent = Agent(address=destination_vip, enable_store=False)
+                agent = Agent(address=destination_vip,
```

Comment:
use the build_agent function from volttron.vip.agent.utils pass the same arguments that you do here.

Fix the event wait in that function while you are at it :)